app/controllers/util/FormAction.scala

```diff
@@ -34,9 +36,19 @@ object FormAction {
     }.headOption.getOrElse(Unknown)
 }
 
-case object Unknown extends FormAction
-case object Add extends FormAction
-case object SaveAndContinue extends FormAction
-case object Continue extends FormAction
+case object Unknown extends FormAction {
+  override val label: String = "Unknown"
```

Comment:
Why you do this?

Result from val label is the same like `toString` method.

toString method from case objects return their name. So I'm not sure about approach and intension here.

Reply:
It is not the same for the Remove case class, because `toString()` adds all the fields from case class. Here, we want to have the class name only.